app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -277,7 +277,14 @@ function findMissingSemicolons(content) {
       continue
     }
 
-    if (content[i] == "/" && content[i + 1] == "/") {
+    if (content[i] == "/") {
+      if(content[i + 1] == "/" || content[i + 1] == "*") {
```

Comment:
I didn't even know comments blocks like /* were supported.

Reply:
yeah you can paste them into the game! but the compiler already removes comments anyway